pkg/cmd/pipelinerun/describe.go

```diff
@@ -0,0 +1,140 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelinerun
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"io"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+	"text/tabwriter"
+)
+
+const (
+	statusHeader = "STARTED\tDURATION\tSTATUS\n"
+	statusBody = "%s\t%s\t%s\n"
+	resourceHeader = "NAME\tRESOURCE REF\n"
+	resourceBody = "%s\t%s\n"
+	paramHeader = "NAME\tVALUE\n"
+	paramBody = "%s\t%s\n"
+	taskrunHeader = "NAME\tSTARTED\tDURATION\tSTATUS\n"
+	taskrunBody = "%s\t%s\t%s\t%s\n"
+)
+
+func describeCommand(p cli.Params) *cobra.Command {
+	f := cliopts.NewPrintFlags("describe")
+	eg := `
+# Describe a PipelineRun of name 'foo' in namespace 'bar'
+tkn pipelinerun describe foo -n bar
+
+tkn pr desc foo -n bar \n",
```

Comment:
You don't need that \n, it shows verbatim in help, i.e: 

![image](https://user-images.githubusercontent.com/98980/58476971-ab22d980-8152-11e9-99aa-38572ecfcf74.png)